inference-engine/tests/functional/plugin/shared/src/behavior/stress_tests.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/stress_tests.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+
+#define BUG_ALLOCATION_LIMIT 10000
```

Comment:
Maybe we should make it a test parameter?

Reply:
It could be so.